test/statistics-analysis/scripts/run-tests.sh

```diff
@@ -47,6 +47,34 @@ get_pod_name() {
     echo $pod
 }
 
+resolve_deployment() {
+    local name=$1
+    local deployment="${!name}"
+
+    if [ "$name" == "baseline" ]; then
+	return 0
+    fi
+    if [ -z "$deployment" ]; then
+	return 0
+    fi
+
+    if [ -f "$deployment" ]; then
+	return 0
+    fi
```

Comment:
nit, can we perhaps merge these two operators?
```suggestion
    if  [ -z "$deployment" ] || [ -f "$deployment" ]; then
	return 0
    fi
```

Reply:
Updated.